.github/run-benchmark_hf-as-lit-job.py

```diff
@@ -15,7 +15,15 @@ def main():
     print("Starting studio...")
     s.start()
     print("Installing Thunder and dependencies...")
-    s.run("pip install lightning-thunder transformers -f dist/ -U")
+
+    # install from latest commit
+    s.run(
```

Comment:
I think this shall install the version from source especially if added run on PR